packages/react-charting/UnitTests/HeatMapChartUT.test.tsx

```diff
@@ -574,19 +576,22 @@ runTest('_createNewDataSet', () => {
 });
 
 runTest('Skip - timezone related test cases', () => {
-  test('Should return proper data set for date axis with default axis type', () => {
-    const instance = new HeatMapChartBase({
-      data: emptyData,
-      domainValuesForColorScale: [],
-      rangeValuesForColorScale: [],
-    });
-    expect(instance).toBeDefined();
-    const result = instance._createNewDataSet(HeatMapDataDatePoints, '%b/%d', '.2~s', '%b/%d', '.2~s');
-    expect(result.xAxisPoints[0]).toEqual('Fri Apr 03 2020 05:30:00 GMT+0530 (India Standard Time)');
-    expect(result.yAxisPoints[0]).toEqual('Tue Mar 03 2020 05:30:00 GMT+0530 (India Standard Time)');
-  });
-
-  test('Should return proper aria-label for date xPoint and yPoint', () => {
+  conditionalTest(isTimezoneSet(Timezone.UTC))(
+    'Should return proper data set for date axis with default axis type',
```

Comment:
What about date object based tests in LinChart, Vertical Bar Chart and Vertical Stacked Bar Chart.

Reply:
Needs to be fixed in the next iteration.